rados/ioctx.go

```diff
@@ -19,6 +19,9 @@ import (
 	"unsafe"
 )
 
+// TODO Use enum directly from librados
+const LIBRADOS_OP_FULL_TRY = 64
```

Comment:
1. I have already tried that which I think I spoke with you yesterday, this enum is unnamed so it can't be accessed that way. 
2. We get EDQUOTA only when quota is set on the pool, we don't set quota on the pools so not needed.
3. I have tried FORCE_FULL option it won't be useful in our case as we will still need to log SSDLog writes which would eventually fail. If its not needed then I don't want to use it as
it looks like some forceful option. If and when we update truncate operations from starling side to truncate based on subset of offsets within keys then we can add that.